src/utils/dataFormatters/dataFormatters.ts

```diff
@@ -123,12 +128,13 @@ export const formatSecondsToHours = (seconds: number) => {
 };
 
 export const roundToPrecision = (value: number | string, precision = 0) => {
-    let [digits] = String(value).split('.');
-    if (Number(value) < 1) {
+    const valueAbs = Math.abs(Number(value));
+    let [digits] = String(valueAbs).split('.');
+    if (Number(valueAbs) < 1) {
```

Comment:
In previous code negative values always had more digits length, so rounding didn't work well
"-1".length = 2
"1".length = 1